website/docs/best-practices/how-we-mesh/mesh-4-faqs.md

```diff
@@ -0,0 +1,314 @@
+---
+title: "dbt Mesh FAQs"
+description: "Read the FAQs to learn more about dbt Mesh, how it works, compatibility, and more."
+hoverSnippet: "dbt Mesh FAQs"
+sidebar_label: "dbt Mesh FAQs"
+---
+
+dbt Mesh is a new architecture enabled by dbt Cloud. It allows you to better manage complexity by deploying multiple interconnected dbt projects instead of a single large, monolithic project. It’s designed to accelerate development, without compromising governance.
+
+## Overview of Mesh
+
+<detailsToggle alt_header="How do I implement contracts for my models?">
```

Comment:
As this isn't actually providing any implementation instructions and only defining the feature, I suggest we change the question to
```suggestion
<detailsToggle alt_header="What are model contracts?">
```